build.gradle

```diff
@@ -19,44 +19,38 @@ buildscript {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
         classpath "com.google.guava:guava:$guavaVersion"
         classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
-        classpath 'org.grails.plugins:hibernate5:6.0.4'
+        classpath 'org.grails.plugins:hibernate5:6.1.12'
         classpath "com.google.protobuf:protobuf-java:${protobufVersion}"
         classpath "com.google.protobuf:protobuf-java-util:${protobufVersion}"
-        classpath 'org.grails.plugins:views-gradle:1.0.12'
+        classpath 'org.grails.plugins:views-gradle:1.2.10'
         classpath 'net.linguica.gradle:maven-settings-plugin:0.5'
         classpath "org.fusesource.jansi:jansi:${jansiVersion}"
-        classpath 'org.owasp:dependency-check-gradle:3.2.1'
+        classpath 'org.owasp:dependency-check-gradle:5.3.2.1'
     }
 }
 
-// Workaround suggested in https://github.com/gradle/gradle/issues/778
-org.fusesource.jansi.AnsiConsole.out.println('[init jansi library conflict workaround]')
-
 apply plugin: 'idea'
 apply plugin: 'maven-publish'
 apply plugin: 'org.owasp.dependencycheck'
 
 ext {
     transmartVersion = '17.2-SNAPSHOT'
 
-    assetPipelineVersion = '2.11.6'
-    gradleWrapperVersion = '3.5.1'
-    grailsVersion = '3.2.13'
-    guavaVersion = '19.0'
-    jacksonVersion = '2.10.1'
-    liquibaseVersion = '3.7.0'
-    ojdb7Version = '12.1.0.1'
+    assetPipelineVersion = '2.15.1'
+    grailsVersion = '3.3.11'
+    guavaVersion = '29.0-jre'
+    hibernateVersion = '5.1.9.Final'
+    jacksonVersion = '2.11.0'
+    liquibaseVersion = '3.10.0'
+    ojdb8Version = '12.2.0.1'
     openCsvVersion = '5.0'
-    postgresqlVersion = '42.2.5.jre7'
-    protobufVersion = '3.1.0'
-    quartzVersion = '2.0.13'
+    postgresqlVersion = '42.2.14.jre7'
+    protobufVersion = '3.12.2'
+    quartzPluginVersion = '2.0.13'
+    quartzVersion = '2.3.2'
     springSecurityPluginVersion = '3.2.3'
-    springSecurityVersion = '4.1.4.RELEASE'
-    keycloakVersion = '6.0.1'
-}
-
-task wrapper(type: Wrapper) {
-    gradleVersion = gradleWrapperVersion
+    springSecurityVersion = '4.2.17.RELEASE'
+    keycloakVersion = '8.0.2'
```

Comment:
Why this version? Unfortunately there is a bug introduced in v. 8.x - https://issues.redhat.com/browse/KEYCLOAK-10595 and the application doesn't start. It is fixed in version > 9 (I tested with 9.0.3). Could we use a newer than 8?

Reply:
I had issues with tests with version `10`, nice that version 9 works. I updated the dependency.